app/models/post.rb

```diff
@@ -1,3 +1,23 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  has_many :likes
+  has_many :comments, foreign_key: :post_id
```

Comment:
- [ ] To follow the ERD diagram, kindly remove the `foreing_key`:post_id reference, since it's not necessary to create the relation between each model